aten/src/ATen/native/AdaptiveAveragePooling3d.cpp

```diff
@@ -135,7 +137,7 @@ void adaptive_avg_pool3d_out_cpu_template(
   if (input.ndimension() == 4) {
     output.resize_({sizeD, osizeT, osizeH, osizeW});
 
-    AT_DISPATCH_FLOATING_TYPES_AND_HALF(
+    AT_DISPATCH_FLOATING_TYPES_AND2(kHalf, kBFloat16,
```

Comment:
Only `adaptive_{max, avg}_pool3d` where missing `bfloat16` support, other variants supported. So I updated it. Is that okay?

Reply:
I think this would be out-of-scope for this PR. Probably worth pursuing in another PR.

Reply:
Sure, sent PR #89754. I'll rebase this PR after that is merged. Will that be okay?